cdc/scheduler/internal/v3/replication/replication_manager.go

```diff
@@ -28,6 +30,10 @@ import (
 
 const (
 	checkpointCannotProceed = internal.CheckpointCannotProceed
+
+	defaultSlowTableHeapSize  = 16
```

Comment:
can you set the value of defaultSlowTableHeapSize  with  a little number?

Reply:
ok